js/tabs/maps/F8MapView.js

```diff
@@ -48,19 +48,33 @@ class F8MapView extends React.Component {
 
   render() {
     const {map1, map2} = this.props;
+
+	var map1Name = 'Overview';
+	var map2Name = 'Developer Garage';
+
+  	var readerEmptyListMap1 = <View />;
+  	var readerEmptyListMap2 = <View />;
+  	if(map1 && map2) {
+  		readerEmptyListMap1 = <MapView map={map1} />;
+  		readerEmptyListMap2 = <MapView map={map2} />;
+  		
+  		map1Name = map1.name;
+  		map2Name = map2.name;
+  	}
+	    
     return (
       <View style={styles.container}>
         <ListContainer
           title="Maps"
           backgroundImage={require('./img/maps-background.png')}
           backgroundColor={'#9176D2'}>
           <PureListView
-            title={map1.name}
-            renderEmptyList={() => <MapView map={map1} />}
+            title={map1Name}
+            renderEmptyList={() => readerEmptyListMap1}
```

Comment:
`<MapView />` gracefully handles the case when map is empty, we don't need `readerEmptyListMap1/2`.